src/Alert/Alert.stories.js

```diff
@@ -0,0 +1,177 @@
+              sectionFn: () => (
+                <Alert type="success" title={title} noIcon={noIcon}>
+                  {message}
+                </Alert>
+              ),
+              options,
+            },
+          ],
+        },
+      ],
+    };
+  })
+  .addWithChapters("Warning alert", () => {
+    const title = text("Title", "");
+    const message = text("Message", "The quick, brown fox jumps over a lazy dog.");
+    const noIcon = boolean("No icon", false);
+    return {
+      info:
+        "Use in cases when you need to inform users about a potentially dangerous situation in their trip and it requires some action from them. However, if the issue requires immediate attention, use critical alert instead. Visit Orbit.Kiwi for more detailed guidelines.",
+      chapters: [
+        {
+          sections: [
+            {
+              sectionFn: () => (
+                <Alert type="warning" title={title} noIcon={noIcon}>
+                  {message}
+                </Alert>
+              ),
+              options,
+            },
+          ],
+        },
+      ],
+    };
+  })
+  .addWithChapters("Critical alert", () => {
+    const title = text("Title", "");
+    const message = text("Message", "The quick, brown fox jumps over a lazy dog.");
+    const noIcon = boolean("No icon", false);
+    return {
+      info:
+        "Use when something is blocking users from continuing or when some issue needs to be resolved immediately. A critical alert should provide some form of solution for their problem. If something is important for users to solve as soon as possible, automatic open of a modal window is worthy of considering. Visit Orbit.Kiwi for more detailed guidelines.",
+      chapters: [
+        {
+          sections: [
+            {
+              sectionFn: () => (
+                <Alert type="critical" title={title} noIcon={noIcon}>
+                  {message}
+                </Alert>
+              ),
+              options,
+            },
+          ],
+        },
+      ],
+    };
+  })
+  .addWithChapters("Playground", () => {
+    const typeOptions = {
+      info: "info",
+      success: "success",
+      warning: "warning",
+      critical: "critical",
+    };
+    const type = select("Type", typeOptions, "info");
+    const title = text("Title", "You can change the title by changing the Title knob");
+    const message = text("Message", "Also you can change the message by changing the Message knob");
+    const closable = boolean("Closable", false);
+    const noIcon = boolean("No icon", false);
+    const source = select("Icon", Object.keys(Icons));
+    return {
+      info:
+        "You can try all possible configurations of this component. However, check Orbit.Kiwi for more detailed design guidelines.",
+      chapters: [
+        {
+          sections: [
+            {
+              sectionFn: () => (
+                <Alert
+                  type={type}
+                  Icon={Icons[source]}
```

Comment:
Why this prop is with capital letter in the start ? 🤔

Reply:
It's the only case when it needs to be capitalized.
```
const { Icon,... } = props;

return (
   <Icon ... />
)
```
If we use `icon` with a small letter on the start, we get an error:
`If you meant to render a React component, start its name with an uppercase letter.`